client/src/pages/CommunityPage/CommunityPagePeople.js

```diff
@@ -23,25 +26,36 @@ function pillClick(val) {
 
 const handleClick = e => {};
 
-// Page or
 const CommunityPage = props => {
+  // State Hooks
   const [profile, setProfile] = useState([]);
+  const [states, setStates] = useState([]);
+  const [cities, setCities] = useState([]);
 
   // Getting profile list
   useEffect(() => {
     const fetch = async () => {
-      const res = await axios.get("http://127.0.0.1:8000/profile");
+      const res = await axios.get(`${BASE_URL}/profile`);
       setProfile(res.data);
-      // console.log(res);
     };
     fetch();
   }, []);
 
-  //random number between 1 and 5
+  // Getting US States
+  useEffect(() => {
+    const fetch = async () => {
+      const res = await axios.get(`${BASE_URL}/location/states`);
+      setStates({ USStates: res.data });
+    };
+    fetch();
+  }, []);
```

Comment:
Can you put this in a service?

Reply:
All interactions with the API should be through a service

Reply:
@MilesHamilton

Reply:
Removed CommunityPagePeople from commit.